Enhanced Log/enhancedsyslogsettings.txt

```diff
@@ -87,76 +87,77 @@ Other Highlight Color=Andere Hervorhebungsfarbe
 
 :enhanced_log_color_login_plug:
-> Color to use for 'Login' highlighting.  You can use the color name or number.  The color number begins with a '#'.
+> Zu verwendende Farbe für die Hervorhebung von 'Login'.  Es kann der Name der Farbe oder der Farbcode verwendet werden.  Der Farbcode beginnt mit einem '#'.
 :end
 
 :enhanced_log_color_emhttp_plug:
-> Color to use for 'emhttp' highlighting.  You can use the color name or number.  The color number begins with a '#'.
+> Zu verwendende Farbe für die Hervorhebung von 'emhttp'.  Es kann der Name der Farbe oder der Farbcode verwendet werden.  Der Farbcode beginnt mit einem '#'.
 :end
 
 :enhanced_log_color_other_plug:
-> Color to use for 'Other' highlighting.  You can use the color name or number.  The color number begins with a '#'.
+> Zu verwendende Farbe für die Hervorhebung von 'Weitere'.  Es kann der Name der Farbe oder der Farbcode verwendet werden.  Der Farbcode beginnt mit einem '#'.
 :end
 
 :enhanced_log_custom_plug:
-> Enter any custom events you want highlighted in the log.  Any custom events you enter will be processed after the built in events.
-> The events are entered as: "search string", "event"
-> Case is ignored in the search string and wild cards are allowed.
-> Examples:
-> "found.*chip" will match any string with 'found' and 'chip' with any text in between,
-> "(spinup|spindown)" will match any string with either 'spinup' or 'spindown'. 
-> The search string and event are quoted, and each line contains one match string and event combination.  Errors in the strings can cause php errors.
-> Events are: "error", "minor issues", "lime tech", "array", "system", "file system", "drive related", "network", "login", "emhttp", "other".
+> Gib ein benutzerdefiniertes Ereignis ein welches du im Protokoll hervorheben willst.  Jedes benutzerdefiniertes Ereignis wird im Anschluss an die systemintegrierten Ereignisse angewendet.
+> Der Syntax für benutzerdefinierte Ereignisse ist: "zu suchende Zeichnefolge", "Ereignis"
+> Groß-/Kleinschreibung wird ignioriert, Platzhalter sind erlaubt.
+> Beispiele:
+> "found.*chip" wird jede Zeichenfolge finden, die 'found' und 'chip' enthält mit beliebigem Text dazwischen,
+> "(spinup|spindown)" wird jede Zeichenfolge finden, die entweder 'spinup' oder 'spindown' enthält.
+> Die zu suchende Zeichnefolge und das Ereignis stehen in Anführungszeichen und jede Zeile enthält eine Kombination von Treffer und Ereignis. Fehler in den zu suchenden Zeichnefolgen können PHP Fehler verursachen.
+> Ereignisse sind: "error", "minor issues", "lime tech", "array", "system", "Dateisystem", "drive related", "network", "login", "emhttp", "other".
 :end
 
 :enhanced_log_custom_buttons_plug:
-> Click the **Apply** button to save your custom events.  Click **Reset** to
-> undo any changes you make before you have saved the custom events. Click **Done** to exit this page.
+> Klick auf **Übernehmen** um die benutzerdefinierten Ereignisse zu speichern. Klick auf **Abbrechen** um
```

Comment:
Statt "Ereignis" -> "Vorgang"?

Reply:
hmmm, "event" ist korrekt übersetzt eigentlich Ereignis... (in Anlehnung an eventlog => Ereignisprotokoll)